store/src/main/java/org/apache/rocketmq/store/schedule/ScheduleMessageService.java

```diff
@@ -113,7 +115,7 @@ public long computeDeliverTimestamp(final int delayLevel, final long storeTimest
     public void start() {
         if (started.compareAndSet(false, true)) {
             super.load();
-            this.timer = new Timer("ScheduleMessageTimerThread", true);
+            this.scheduledExecutorService = new ScheduledThreadPoolExecutor(32, new DefaultThreadFactory("ScheduleMessageTimerThread"));
```

Comment:
IMO, it would be better if the thread pool core size is configureable or just use `Runtime.getRuntime().availableProcessors()`

Reply:
agree

Reply:
Ok, I will deal with it later